ospd_openvas/wrapper.py

```diff
@@ -210,7 +226,8 @@ def __init__(self, certfile, keyfile, cafile):
         """ Initializes the ospd-openvas daemon's internal data. """
 
         super(OSPDopenvas, self).__init__(certfile=certfile, keyfile=keyfile,
```

Comment:
Unrelated to this PR: Could be simplified with python 3
```suggestion
        super().__init__(certfile=certfile, keyfile=keyfile,
```